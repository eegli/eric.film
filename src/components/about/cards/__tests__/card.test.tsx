import { shallow } from 'enzyme';
import Card from '../card.component';

describe('<Card />', () => {
  it('renders three <Foo /> components', () => {
    const wrapper = shallow(<Card />);
    expect(wrapper).toMatchSnapshot();
  });
});
