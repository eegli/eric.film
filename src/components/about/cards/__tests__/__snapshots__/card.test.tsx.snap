// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About - card renders component 1`] = `
<Fragment>
  <cardstyles__CardHeader>
     meet the amazing team 
  </cardstyles__CardHeader>
  <cardstyles__CardContainer>
    <cardstyles__Card>
      <cardstyles__CardPicture>
        <source
          srcSet="/static/img/about/about-2.webp"
          type="image/webp"
        />
        <img
          alt="logo"
          src="/static/img/about/about-2.jpg"
        />
      </cardstyles__CardPicture>
      <cardstyles__CardTitle>
        eric
      </cardstyles__CardTitle>
      <cardstyles__CardDescription>
        <span>
          FOUNDER
        </span>
        <p>
          eric has been with us from the start. he is the founding father of eric.film and oversees all business activities as well as HR-related topics. Main responsibilities:
        </p>
        <ul>
          <li>
            project management
          </li>
          <li>
            photography
          </li>
        </ul>
        <SocialBar />
      </cardstyles__CardDescription>
    </cardstyles__Card>
    <cardstyles__Card>
      <cardstyles__CardPicture>
        <source
          srcSet="/static/img/about/about-1.webp"
          type="image/webp"
        />
        <img
          alt="logo"
          src="/static/img/about/about-1.jpg"
        />
      </cardstyles__CardPicture>
      <cardstyles__CardTitle>
        eric
      </cardstyles__CardTitle>
      <cardstyles__CardDescription>
        <span>
          LEAD DEVELOPER
        </span>
        <p>
          eric is our self-taught javascript engineer. he mostly talks about "react" but as long as our website looks good, we don't really care. Main responsibilities:
        </p>
        <ul>
          <li>
            eric.film development
          </li>
          <li>
            blogging
          </li>
        </ul>
        <SocialBar />
      </cardstyles__CardDescription>
    </cardstyles__Card>
    <cardstyles__Card>
      <cardstyles__CardPicture>
        <source
          srcSet="/static/img/about/about-3.webp"
          type="image/webp"
        />
        <img
          alt="logo"
          src="/static/img/about/about-3.jpg"
        />
      </cardstyles__CardPicture>
      <cardstyles__CardTitle>
        eric
      </cardstyles__CardTitle>
      <cardstyles__CardDescription>
        <span>
          INTERN
        </span>
        <p>
          eric has bachelor of arts in communication science so we let him do all the client talking. other than that, we only really see him at after-work drinks. Main responsibilities:
        </p>
        <ul>
          <li>
            none
          </li>
        </ul>
        <SocialBar />
      </cardstyles__CardDescription>
    </cardstyles__Card>
  </cardstyles__CardContainer>
</Fragment>
`;
